src/explorer.ts

```diff
@@ -20,13 +20,28 @@ export function createKubernetesResource(kind: kuberesources.ResourceKind, id: s
 }
 
 function getIconForPodStatus(status: string): vscode.Uri {
-    if (status === "running" || status === "completed") {
+    if (status === "Running" || status === "Completed") {
         return vscode.Uri.file(path.join(__dirname, "../../images/runningPod.svg"));
     } else {
         return vscode.Uri.file(path.join(__dirname, "../../images/errorPod.svg"));
     }
 }
 
+function parsePodLine(podInfo: string[]): any[] {
+    const headers = podInfo.shift();
+    const parsedHeaders = headers.toLowerCase().replace(/\s+/g, '|').split('|');
+    const pods = [];
+    podInfo.map((line) => {
+        const podObject = {};
+        const bits = line.replace(/\s+/g, '|').split('|');
+        bits.map((columnValue, index) => {
```

Comment:
Can this be done with `forEach`?  `map` implies the function is performing a transformation and you are interested in the return values of the transformation, whereas this is being done purely for its side-effects (to populate the `podObject`).

Reply:
sounds good.